src/components/tables/Table.tsx

```diff
@@ -123,6 +125,12 @@ export class Table extends React.Component<ITableProps, {}> {
     }
   }
 
+  componentWillUpdate(tableCompositeState: any) {
+    if (this.props.onWillUpdate && !(JSON.stringify(tableCompositeState.actions) === JSON.stringify(this.props.actions))) {
```

Comment:
same same but instead of !(JSON.stringify(tableCompositeState.actions) === JSON.stringify(this.props.actions)) you could use (JSON.stringify(tableCompositeState.actions) !== JSON.stringify(this.props.actions))